src/input.service.ts

```diff
@@ -475,7 +480,7 @@ export class InputService {
    * a connected gamepad somewhere.
    */
   private pollGamepad(now: number) {
-    const rawpads = navigator.getGamepads().filter(pad => !!pad); // refreshes all checked-out gamepads
+    const rawpads = Array.from(navigator.getGamepads()).filter(pad => !!pad); // refreshes all checked-out gamepads
```

Comment:
http://kangax.github.io/compat-table/es6/#test-Array_static_methods_Array.from,_array-like_objects

Doesn't seem to support IE which iirc we want to support.